reference/5.1/PSDesiredStateConfiguration/Get-DscLocalConfigurationManager.md

```diff
@@ -22,38 +23,108 @@ Get-DscLocalConfigurationManager [-CimSession <CimSession[]>] [-ThrottleLimit <I
 ```
 
 ## DESCRIPTION
-The **Get-DscLocalConfigurationManager** cmdlet gets Local Configuration Manager (LCM) settings, or meta-configuration, and the states of LCM for the node.
-Specify computers by using Common Information Model (CIM) sessions.
-If you do not specify a target computer, the cmdlet gets the configuration settings from the local computer.
+
+The `Get-DscLocalConfigurationManager` cmdlet gets LCM settings, or meta-configuration, and the
+states of LCM for the node. Specify computers by using Common Information Model (CIM) sessions. If
+you do not specify a target computer, the cmdlet gets the configuration settings from the local
+computer.
 
 ## EXAMPLES
 
 ### Example 1: Get LCM settings for the local computer
+
+```powershell
+Get-DscLocalConfigurationManager
 ```
-PS C:\> Get-DscLocalConfigurationManager
+
+```output
```

Comment:
Output should be capitalized. Since it is not a "supported" code language, OPS just copies the string into the header of the code box. We want it to be capitalized on the web page so it must be so here.

Reply:
fixed